test/integration/targets/mount_facts/aliases

```diff
@@ -0,0 +1,4 @@
+shippable/posix/group1
+context/target
+needs/root
+skip/docker
```

Comment:
This test should be marked `destructive` since it makes changes to the system.

Reply:
Thanks, good catch. I had written this test non-destructive initially.